modules/nf-core/bwa/mem/tests/main.nf.test

```diff
@@ -170,4 +174,103 @@ nextflow_process {
         }
 
     }
+
+    test("Paired-End - no fasta") {
+
+        setup {
+            run("BWA_INDEX") {
+                script "../../index/main.nf"
+                process {
+                    """
+                    input[0] = [
+                        [id: 'test'],
+                        file(params.modules_testdata_base_path + 'genomics/sarscov2/genome/genome.fasta', checkIfExists: true)
+                    ]
+                    """
+                }
+            }
+        }
+
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'test', single_end:false ], // meta map
+                    [
+                        file(params.modules_testdata_base_path + 'genomics/sarscov2/illumina/fastq/test_1.fastq.gz', checkIfExists: true),
+                        file(params.modules_testdata_base_path + 'genomics/sarscov2/illumina/fastq/test_2.fastq.gz', checkIfExists: true)
+                    ]
+                ]
+                input[1] = BWA_INDEX.out.index
+                input[2] = [[id: 'test'],[]]
```

Comment:
```suggestion
                input[2] = [[:],[]]
```

Reply:
I can agree with this one. I don't think it really matter 🤷🏻

Reply:
Yep, just wanted to make sure the meta content wasn't required for anything. I'll stop picking nits now